kuma/users/tests/test_stripe_subscription.py

```diff
@@ -13,18 +15,44 @@
 from kuma.core.utils import safer_pyquery as pq
 from kuma.users.models import User
 
-
 from . import user
 
 
-def mock_retrieve_stripe_subscription_info(user):
-    return {
-        "next_payment_at": "",
-        "brand": "MagicCard",
-        "expires_at": "",
-        "last4": 4242,
-        "zip": 1337,
-    }
+@dataclass
+class StripeCustomerSource:
+    object: str
+    brand: str
+    exp_month: int
+    exp_year: int
+    last4: int
+
+    def get(self, key, default=None):
```

Comment:
This right here is what I switched to dataclasses instead of the `SimpleNamespace`. The mock of the customer needs a `get` method.